www/src/pages/de/introduction.md

```diff
@@ -0,0 +1,43 @@
+
+Du hast vielleicht bemerkt, dass der T3-Stack viele Bestandteile hat. Das ist so gewollt. Tausche die Teile aus, wenn du sie brauchst - der Stack ist im Kern modular :)
+
+## Also... was ist create-t3-app? Ein Template?
+
+Irgendwie? `create-t3-app` ist eine CLI, die von erfahrenen T3-Stack-Entwicklern entwickelt wurde, um die Einrichtung einer modularen T3-Stack-App zu vereinfachen. Das bedeutet, dass jeder Bestandteil optional ist und das "Template" auf der Grundlage deiner spezifischen Anforderungen erstellt wird.
+
+After countless projects and many years on this tech, we have lots of opinions and insights. We've done our best to encode them into this CLI.
+Nach unzähligen Projekten und vielen Jahren auf diesem Gebiet haben wir viele Meinungen und Erkenntnisse. Wir haben unser Bestes getan, um sie in dieser CLI zu verwirklichen.
+
+Das hier ist **KEINE** allumfassende Vorlage. Wir **erwarten**, dass du deine eigenen Bibliotheken einbringst, die die Anforderungen **DEINER** Anwendung erfüllen. Wir wollen zwar keine Lösungen für spezifischere Probleme wie Zustandsverwaltung und Deployment vorschreiben, aber wir [haben hier einige Empfehlungen aufgelistet](/de/other-recs).
+
+## T3 Grundsätze
+
+Das Projekt wurde nach _unseren Vorstellungen_ gestaltet. Wir teilen eine Reihe an Überzeugungen für das Bauen von Web-Applikationen und nutzen diese als Basis für unsere Entscheidungen.
+
+### Probleme lösen
+
+Es ist leicht, in die Falle zu tappen, "alles hinzuzufügen" - das wollen wir ausdrücklich nicht tun. Alles, was zu `create-t3-app` hinzugefügt wird, sollte ein spezifisches Problem lösen, das innerhalb der enthaltenen Kerntechnologien existiert. Das bedeutet, dass wir keine Technologien wie Zustandsbibliotheken (`zustand`, `redux`) hinzufügen werden. Wir werden jedoch Dinge wie NextAuth.js, Prisma und tRPC integrieren.
+
+### Modern und verantwortungsbewusst
+
+Wir lieben unsere modernen Technologien. Die Geschwindigkeit und, ehrlich gesagt, der Spaß, der aus dem neuen Zeug entsteht, ist wirklich cool. Wir denken, dass es wichtig ist, verantwortungsvoll zu sein und riskantere Technologien in den weniger riskanten Teilen zu verwenden. Das bedeutet, dass wir nicht ⛔️ auf riskante neue Datenbanktechnologien setzen würden (SQL ist toll!). Aber wir setzen gerne ✅ auf tRPC, da es sich nur um Funktionen handelt, die trivial zu verlagern sind.
+
+### Typsicherheit ist nicht optional
+
+Das erklärte Ziel von `create-t3-app` ist es, den schnellsten Weg zu einer neuen vollwertigen, **typsicheren** Webanwendung zu bieten. Wir nehmen die Typsicherheit, da sie unsere Produktivität verbessert und uns hilft, weniger Fehler auszuliefern. Jede Entscheidung, die die Typsicherheit von `create-t3-app` gefährdet, sollte in einem anderen Projekt getroffen werden.
```

Comment:
```suggestion
Das erklärte Ziel von `create-t3-app` ist es, den schnellsten Weg zu einer neuen, **typsicheren** Full-Stack Webanwendung zu bieten. Wir nehmen die Typsicherheit ernst, da sie unsere Produktivität verbessert und uns hilft, weniger Fehler auszuliefern. Jede Entscheidung, die die Typsicherheit von `create-t3-app` gefährdet, sollte in einem anderen Projekt getroffen werden.
```

not sure what the correct translation of fullstack is, but it's not "vollwertig"